src/peer/socket.rs

```diff
@@ -99,103 +91,107 @@ async fn retainer<S: Signer>(
 
         u.set_query(Some(token.as_str()));
 
+        log::info!("connecting to relay {:?}", &u.domain());
         let (ws, _) = match tokio_tungstenite::connect_async(&u).await {
             Ok(v) => v,
             Err(err) => {
-                log::trace!(
+                log::error!(
```

Comment:
I think it's better to show connection instability by default. this can give an early sign that a certain relay is in a bad condition, and give us a chance to choose to fix it or drop it from the relay lists.

If it's hidden as debug (not everyone run with debug flags) we can end up in the situation that is failing (for any reason) and we have no indication that the peer is struggling to get a stable connection.